.travis.yml

```diff
@@ -110,7 +110,7 @@ script:
   # Generate documentation and run doctests
   - PYTHONPATH=$PWD make -C docs html doctest
   # Run javascript tests
-  - grunt test --verbose --force
+  - grunt test --verbose
```

Comment:
with the `--force` if the js_tests the test runs wouldn't pass unlike before when it was just noted but didn't actually fail the travis/buildbot runs.